Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -345,18 +345,26 @@ def create_pull_request(cls, repository, args, branch_point, callback=None, unbl
         remote_repo = repository.remote(name=source_remote)
         if isinstance(remote_repo, remote.GitHub):
             if issue and issue.redacted and args.remote is None:
-                print("Your issue is redacted, diverting to a secure, non-origin remote you have access to.")
+                print('{} is considered the primary issue for your pull request'.format(issue.link))
```

Comment:
Is there a concept of primary and secondary issues?

Reply:
Yes, it's basically part of supporting both radar and bugzilla. https://github.com/WebKit/WebKit/blob/a203e214d6b8191acdc2aeb7a7430ff5f761cdce/Tools/Scripts/libraries/webkitscmpy/webkitscmpy/commit.py#L283 is the code involved here. "Primary" really means "the first one we find" in most cases, but there are exceptions to that, like a commit title that references a bug.